awx/ui/src/components/Lookup/Lookup.js

```diff
@@ -54,14 +54,15 @@ function Lookup(props) {
   } = props;
   const [typedText, setTypedText] = useState('');
   const debounceRequest = useDebounce(onDebounce, 1000);
-
   useField({
     name: fieldName,
     validate: (val) => {
-      if (!multiple && !val && typedText && typedText !== '') {
-        return t`That value was not found. Please enter or select a valid value.`;
-      }
-      return validate(val);
+      setTimeout(() => {
```

Comment:
Use [`setTimeout`](https://github.com/formium/formik/issues/2083#issuecomment-694265824) to avoid race condition between formik validator and React hooks throwing validation errors.

Reply:
I don't think this is necessary.  I think the issue is that we are calling `setTouched` inside the `onChange` call [here](https://github.com/ansible/awx/pull/11107/files#diff-edaa37419860b1661e289b954b771a9f5e10dfc50976c26d459f193ecc2221e4R253) when in reality we only want to validate that look up(never really, because it isn't required anywhere) when we blur from the field, rather on onChange.